dev/io.openliberty.checkpoint_fat/publish/servers/xmlBindingServer/server.xml

```diff
@@ -0,0 +1,26 @@
+<!-- 
+ (C) Copyright IBM Corporation 2017.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<server description="Sample Servlet server">
+    <featureManager>
+        <feature>jsp-2.3</feature>
```

Comment:
I was expecting jaxb-2.2 feature to be pulled in here

Reply:
resolved